MANIFEST.in

```diff
@@ -1,5 +1,7 @@
 include dash_player/dash_player.min.js
 include dash_player/dash_player.dev.js
+include dash_player/async-dashplayer.js
+include dash_player/async-dashplayer.js.map
```

Comment:
```suggestion
include dash_player/*.js
include dash_player/*.js.map
```
Thanks for calling this out. The `.dev.js` file no longer exists, but the better thing to do is just include all `.js` an `.js.map` files.

There's a good argument to be made that we don't actually want to distribute the sourcemaps, as they can be 3/4 of the total package bytes! The only problem with that is all the bundles reference their sourcemaps at the end of the file like:

https://github.com/plotly/dash-player/blob/079bb4652323d40e1655d364223404bd5350985a/dash_player/async-dashplayer.js#L2

so if we don't include sourcemaps then you get console warnings like you see in the dash docs (where we removed all the sourcemaps to reduce the slug size):

<img width="922" alt="Screenshot 2023-03-08 at 16 18 36" src="https://user-images.githubusercontent.com/2678795/223852531-db540fbf-05db-4b23-9f92-1e2798ef5ff4.png">

So ultimately what we want to do is create a build process that strips those `sourceMappingURL` comments from the bundles so we can omit them and still have a clean console, but they're still available for developers who build the component locally and want to debug.